app/views/layouts/application.html.erb

```diff
@@ -5,6 +5,7 @@
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <%= csrf_meta_tags %>
+  <%= include_gon %>
```

Comment:
I removed that line. I used gon initially but changed to just use the erb template code.